packages/dashboard-frontend/src/containers/Loader/Factory/Steps/Fetch/Devfile/index.tsx

```diff
@@ -341,9 +341,9 @@ class StepFetchDevfile extends AbstractLoaderStep<Props, State> {
           <ExpandableWarning
             textBefore="Looking for a Devfile in the git repository failed."
             errorMessage={helpers.errors.getMessage(error)}
-            textAfter="The git provider may not be supported by Eclipse Che (or the git server hasn't been configured
-            appropriately). Note that the git cloning of the repository will succeed if it's public or it's private
-            and the developer git credentials secret has been configured."
+            textAfter="The git provider is either not supported or has not been configured correctly.
+            Note that the git cloning of the public repository will succeed.
```

Comment:
Just for me to understand this sentence a bit better: "Note that the git cloning of the public repository will succeed."
The application does not know if this particular repository public or private, or does it?
The way I understand it is this is a general note for all possible repositories.
Is this correct?

Reply:
> The application does not know if this particular repository public or private, or does it?

It doesn't know at the current step. It discovers it later.

Reply:
> The way I understand it is this is a general note for all possible repositories.
Is this correct?

yes, correct